src/main/java/creativei/service/impl/FollowUpServiceImpl.java

```diff
@@ -35,30 +35,26 @@ public FollowUp updateFollowUp(FollowUp followUp) throws InvalidParamRequest {
     }
 
     private Boolean validateFollowUp(FollowUp followUp) {
-        Boolean []cond=new Boolean[5];
-        Arrays.fill(cond,false);
-        if (((followUp.getCaseIndex() == null) && (cond[0]=true)) | (followUp.getFollowUpDate() == null&& (cond[1]=true)) | (followUp.getType() == null&& (cond[2]=true)) | (followUp.getStatus() == null && (cond[3]=true)) | (followUp.getSubStatus() == null&& (cond[4]=true))) {
-            for(int i=0;i<cond.length;i++){
-                if(cond[i]==true){
-                    switch (i){
-                        case 0:
-                            logger.error("CaseIndex field does not have appropirate value");
-                            break;
-                        case 1:
-                            logger.error("FollowUp Date field does not have appropirate value");
-                            break;
-                        case 2:
-                            logger.error("FollowUp Type field does not have appropirate value");
-                            break;
-                        case 3:
-                            logger.error("FollowUp Status field does not have appropirate value");
-                            break;
-                        case 4:
-                            logger.error("Follow Up SubStatus field does not have appropirate value");
-                            break;
-                    }
-                }
-            }
+
+        if (followUp.getCaseIndex() == null) {
+            logger.error("CaseIndex field does not have appropirate value");
```

Comment:
Error message should be precise. As we are just checking for null. You should say- "CaseIndex is null"

Reply:
Error messages changed in commit 0d8af9c